html/index.html

```diff
@@ -0,0 +1,223 @@
+      <div class="first-section-container">
+        <h2 class="open-intro-heading">"Hello! Connect, Learn & Grow"</h2>
+        <h1 class="open-heading">
+          ceos international community global summit 2021
+        </h1>
+        <div class="first-section-para">
+          <p>
+            A three-day event to learn from the best in their respective
+            Industries! CEOs Internatioal Community Global Summit'21 brings
+            together successful entrepreneurs, venture capitalists, angel
+            investors and advisors to teach you the tricks of the trade, offer
+            advice, tips and tools for aspiring entrepreneurs as they launch
+            their business.
+          </p>
+        </div>
+        <div class="event-details">
+          <h3><i class="far fa-calendar-alt"></i> 2021.10.4(MON) ~ 6(WED)</h3>
+          <p><i class="far fa-clock"></i> 19:00 ~ 06:00 (UTC+1)</p>
+          <p class="venue">
+            <i class="fas fa-map-marker-alt"></i> M.K.O Abiola National Stadium
+            Abuja, Nigeria
+          </p>
+        </div>
+      </div>
+    </section>
+
+    <section class="second-section">
+      <div class="second-section-container">
+        <h2 class="second-section-heading">Main Program</h2>
+        <hr />
+        <div class="program-list">
+          <div class="lecture flex-box">
+            <img class="icons" src="/assets/program1.png" alt="program-icon" />
+            <h2>Lecture</h2>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Deserunt,
+              veritatis!.
+            </p>
+          </div>
+
+          <div class="startup-block flex-box">
+            <img class="icons" src="/assets/program2.png" alt="program-icon" />
+            <h2>Building Blocks</h2>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Deserunt,
+              veritatis!.
+            </p>
+          </div>
+
+          <div class="growth-hacks flex-box">
+            <img class="icons" src="/assets/program3.png" alt="program-icon" />
+            <h2>Growth Hacks</h2>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Deserunt,
+              veritatis!.
+            </p>
+          </div>
+
+          <div class="forum flex-box">
+            <img class="icons" src="/assets/program5.png" alt="program-icon" />
+            <h2>Forum</h2>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Deserunt,
+              veritatis!.
+            </p>
+          </div>
+        </div>
+        <div class="button-container">
+          <a href="#">See the whole program</a>
+          <button type="submit" class="program-button">
+            Join CIC Global Summit 2021
+          </button>
+        </div>
+      </div>
+    </section>
+    <div class="section-speakers-heading">
+      <h2>Featured Speakers</h2>
+      <hr />
+    </div>
+
```

Comment:
- [ ] From html semantics, the whole section should be wrapped in an `article` tag